src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,95 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__bottom {
+    height: 60px;
+    background-color: #f9f9f9;
+    display: flex;
+    align-items: center;
+    padding-left: 55px;
+  }
+
+  &__columns-wrapper {
+    display: flex;
+    justify-content: space-between;
+    padding: 55px 66px 28px 61px;
+    flex-wrap: wrap;
+    gap: 43px;
+  }
+
+  &__rights {
+    font-family: "Avenir Next";
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 20px;
+
+    color: #939393;
+  }
+
+  &__column {
+    display: flex;
+    flex-direction: column;
+    width: 350px;
+    gap: 14px;
+  }
+
+  &__column-title {
```

Comment:
it should be nested

Reply:
https://github.com/igorizviekov/light-up/pull/7/commits/e1b8f39f5217f487124296a507c3e31c860d8306